mne/viz/tests/test_topo.py

```diff
@@ -77,6 +77,11 @@ def test_plot_topo():
     # Show topography
     evoked = _get_epochs().average()
     plot_evoked_topo(evoked)  # should auto-find layout
+    # Test jointplot
+    evoked.plot_joint()
+    evoked.plot_joint(title='test', ts_args=dict(spatial_colors=True),
+                      topomap_args=dict(colorbar=True, times=[0.]))
```

Comment:
you should update the plot_evoked_erf_erp.py example to show case this new feature